toucan_connectors/snowflake/snowflake_connector.py

```diff
@@ -128,26 +127,47 @@ def _get_warehouses(self) -> List[str]:
             if 'name' in warehouse
         ]
 
+    def _execute_query(self, cursor, query: str, query_parameters: Dict):  # pragma: no cover
+        """Executes `query` against Snowflake's client and retrieves the
+        results. This function exists to reduce the amount of unrelated logic to
+        data retrieval in `_retrieve_data`.
```

Comment:
```suggestion
        results. 
```
Ca ne me parait pas nécessaire de mettre ce commentaire. En général c'est pour ca qu'on découpe en fonction.
A la limite tu peux expliquer pourquoi il y a plusieurs manières de récupérer les données.

Reply:
Fixé aussi !